Automattic/sensei
# Adapt Sensei Core to the Sensei Co-Teachers functionality.

### Changes proposed in this Pull Request
* Replace check for author with check for `edit_post` capability of the lesson for Quizzes.

### Testing instructions
* Create a lesson.
* Confirm you can get a response from the `sensei-internal/v1/lesson-quiz/3033?context=edit&_locale=user` API (you can open the editor for the lesson and it will do the request for you).
* With **another user** confirm you get a permission error when accessing the `sensei-internal/v1/lesson-quiz/3033?context=edit&_locale=user` API.

### New hooks
- `sensei_filter_module_terms_by_owner`: Filter that allows to alter the result of the module terms when being filtered checking for the owner.
- `sensei_teacher_meta_box_coteachers_feature`: Action that is expected to render the coteachers meta box content.


### Screenshot / Video
TBD
